scripts/scil_evaluate_bundles_individual_measures.py

```diff
@@ -118,7 +157,39 @@ def main():
                     output_measures_dict[measure_name] = []
                 output_measures_dict[measure_name].append(
                     measure_dict[measure_name])
-
+    # add set stats if user wants
+    if args.group_statistics:
+        num_of_bundles = len(bundles_references_tuple_extended)
+        # length and span are weighted by streamline count
+        set_total_length = np.sum(
+            np.multiply(output_measures_dict['avg_length'],
+                        output_measures_dict['streamlines_count']))
+        set_total_span = np.sum(
+            np.multiply(output_measures_dict['span'],
+                        output_measures_dict['streamlines_count']))
+        set_streamlines_count = \
+            np.sum(output_measures_dict['streamlines_count'])
+        set_avg_length = set_total_length / set_streamlines_count
```

Comment:
I just realize that `set` could be replace by `group` and it would be clearer from an user perspective